#Custom message for publishing the detected object position
#object_loc.msg
int16 ID
geometry_msgs/Point point
geometry_msgs/Point angles
